src/app/pages/unlock.tsx

```diff
@@ -112,8 +119,14 @@ export function Unlock(): JSX.Element {
           isLoading={loading}
           onClick={submit}
         >
-          Unlock
+          Continue
         </PrimaryButton>
+        <Caption textAlign="left">
+          Forgot your password? Unlock your account by{' '}
+          <Link display="inline" fontSize={-1} onClick={() => navigate(RouteUrls.SignIn)}>
```

Comment:
is `-1` a legit value?

Reply:
I am going to confirm this, I haven't worked with these number increments before and tbh was experimenting with it. It did work for 14px but I should confirm it in some docs.